docs/2.8.0/docs/query/pqs-user-guide.rst

```diff
@@ -49,10 +49,15 @@ The historical table below lists the available Early Access releases of the Part
 |               | *Offset Management*, *Querying Patterns*, *Advanced |
 |               | Querying Topics* sections.                          |
 +---------------+-----------------------------------------------------+
+| `2023-09-18`_ | Documentation updated.  Updated command line        |
+|               | options and added information about using           |
+|               | ``--pipeline-filter`` option.                       |
++---------------+-----------------------------------------------------+
```

Comment:
Same suggestion as for 2.7.1:
```suggestion
| `2023-09-18`_ | Documentation updated. Updated command line        |
|               | options and added information about using the          |
|               | ``--pipeline-parties`` option.                       |
```

Reply:
And same note as for 2.7.1: I changed it to --pipeline-parties rather than --pipeline-filter. Is that correct?

Reply:
(Changed it back to --pipeline-filter)